src/__stories__/slideshow-story.tsx

```diff
@@ -45,3 +60,48 @@ Default.args = {
     loop: false,
     withBullets: true,
 };
+
+type WithCarouselContextArgs = {numItems: number};
+
+export const WithCarouselContext: StoryComponent<WithCarouselContextArgs> = ({numItems}) => {
+    return (
+        <CarouselContextProvider>
+            <FixedFooterLayout
+                footer={
+                    <ResponsiveLayout>
+                        <Box paddingY={16}>
+                            <CarouselContextConsummer>
+                                {({goNext, goPrev, bulletsProps}) => (
+                                    <Inline space="between" alignItems="center">
+                                        <ButtonLink onPress={goPrev}>Prev</ButtonLink>
+                                        <PageBullets {...bulletsProps} />
+                                        <ButtonLink onPress={goNext}>Next</ButtonLink>
```

Comment:
```suggestion
                                        <ButtonLink onPress={goNext} bleedRight>Next</ButtonLink>
```

Reply:
should the arrows be hidden in desktop using the context?

Reply:
why? afaik, the arrows are a "replacement" of the swipe mechanism, and that still applies for non-touch devices.
To hide the arrows maybe we should use a separate prop hideArrowsInDesktop?

Reply:
yes, could be a prop to hide this control for non touch devices